app/src/main/java/io/appium/uiautomator2/model/internal/GestureController.java

```diff
@@ -17,36 +17,23 @@
 package io.appium.uiautomator2.model.internal;
 
 import android.graphics.Point;
+import android.view.ViewConfiguration;
 
 import java.lang.reflect.Array;
-import java.lang.reflect.Constructor;
-import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
 
-import io.appium.uiautomator2.utils.ReflectionUtils;
-
-import static io.appium.uiautomator2.utils.ReflectionUtils.getConstructor;
 import static io.appium.uiautomator2.utils.ReflectionUtils.invoke;
-import static io.appium.uiautomator2.utils.ReflectionUtils.method;
 
 public class GestureController {
-    private static final String POINTER_GESTURE_CLASS = "androidx.test.uiautomator.PointerGesture";
 
-    private static Class<?> pointerGestureClass;
-    private static Constructor<?> pointerGestureConstructor;
     private final Object wrappedInstance;
     private final Method performGestureMethod;
+    private final Gestures gestures;
```

Comment:
Codacy found an issue: [Perhaps 'gestures' could be replaced by a local variable.](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=6222243)